controller-test.sh

```diff
@@ -169,6 +201,30 @@ kubectl -n $CONTROLLER_NAMESPACE patch lagoonbuilds.crd.lagoon.sh lagoon-build-$
 sleep 10
 check_lagoon_build lagoon-build-${LBUILD}
 
+echo "==> Trigger a Task using kubectl apply to test dynamic secret mounting"
+
+kubectl -n $NS apply -f test-resources/dynamic-secret-in-task-project1-secret.yaml
+kubectl -n $NS apply -f test-resources/dynamic-secret-in-task-project1.yaml
+kubectl -n $NS patch lagoontasks.crd.lagoon.sh lagoon-advanced-task-example-task-project-1 --type=merge --patch '{"metadata":{"labels":{"lagoon.sh/controller":"'$CONTROLLER_NAMESPACE'"}}}'
```

Comment:
if you add a bump below this like so:
```
kubectl -n $NS patch lagoontasks.crd.lagoon.sh lagoon-advanced-task-example-task-project-1 --type=merge --patch '{"metadata":{"labels":{"bump":"bump"}}}'
```
It will probably pass tests. [the controller predicates only check the `old` label](https://github.com/uselagoon/remote-controller/blob/main/controllers/v1beta1/predicates.go#L210-L217) so the second patch will bump it and the `old` label will now be the right label

Will have to re-write the predicates to account for old and new label changes. But that is a bit more work than this PR requires, so we can address that later on

Reply:
Ah, right, that makes sense! Thanks for the insight.

Reply:
Sweet, that seems to have done it :)